AnacondaRecipes/pytesseract-feedstock
# pytesseract 0.3.10

**The upstream source:**
License: https://github.com/madmaze/pytesseract/blob/v0.3.10/LICENSE
Requirements:
- https://github.com/madmaze/pytesseract#installation
- https://github.com/madmaze/pytesseract/blob/v0.3.10/setup.cfg

**_Actions:_**
1. Skip `py<37` and `win-64` (`tesseract` isn't available)
2. Remove `noarch`
3. Add missing `setuptools` and `wheel` to `host`
4. Add `tesseract`, update pinnings in `run`
5. Update `test/requires`
6. Add image file `eurotext.tif` to `test/files`
7. Add `pip check`
8. Add `test/commands` for testing how `pytesseract` works as a CLI or as a library
9. Add `license_url`
10. Add doc and dev urls



**Links:**
<details>

* [Anaconda recipe feedstock](https://github.com/AnacondaRecipes/pytesseract-feedstock)

* [Update branch: _v0.3.10_](https://github.com/AnacondaRecipes/pytesseract-feedstock/tree/v0.3.10)

* [PyPi](https://pypi.org/project/pytesseract)

* [Diff between upstream conda-forge **main** and aggreagate **feature** branch](https://github.com/conda-forge/pytesseract-feedstock/compare/main...AnacondaRecipes:v0.3.10#files_bucket)

* [Diff between origin aggregate **master** and aggreagate **feature** branch](https://github.com/AnacondaRecipes/pytesseract-feedstock/compare/master...AnacondaRecipes:v0.3.10#files_bucket)

</details>

**Updating the recipe:**

If the recipe needs additional modification the update branch can be modified.

```
git clone -b v0.3.10 git@github.com:AnacondaRecipes/pytesseract-feedstock.git
```

**Anaconda Linter:**
<details>
The following problems have been found:

WARNING: /aggregate/pytesseract-feedstock/recipe/meta.yaml:65: missing_doc_source_url: The recipe is missing a doc_source_url
Warnings were found
</details>
